src/Aspire.AppHost/Properties/launchSettings.json

```diff
@@ -0,0 +1,57 @@
+{
+  "profiles": {
+    "https": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development",
+        "DOTNET_ENVIRONMENT": "Development",
+        "ASPIRE_DASHBOARD_OTLP_ENDPOINT_URL": "https://localhost:21213",
+        "ASPIRE_RESOURCE_SERVICE_ENDPOINT_URL": "https://localhost:22145"
+      },
+      "dotnetRunMessages": true,
+      "applicationUrl": "https://localhost:17047;http://localhost:15161"
+    },
+    "http": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development",
+        "DOTNET_ENVIRONMENT": "Development",
+        "ASPIRE_DASHBOARD_OTLP_ENDPOINT_URL": "http://localhost:19015",
+        "ASPIRE_RESOURCE_SERVICE_ENDPOINT_URL": "http://localhost:20166"
+      },
+      "dotnetRunMessages": true,
+      "applicationUrl": "http://localhost:15161"
+    },
+    "aspire-sql": {
```

Comment:
```suggestion
    "aspire-mssql": {
```

Reply:
This is to follow the consistent nomenclature of the database type adopted in this repo